cassandra/src/test/java/brave/cassandra/ITTracing.java

```diff
@@ -116,13 +154,65 @@ void execute(Function<Session, BoundStatement> statement) {
     }
   }
 
-  void executeTraced(Function<Session, Statement> statement) {
-    CassandraClientTracing withPropagation = CassandraClientTracing.newBuilder(tracing)
-        .propagationEnabled(true).build();
-    try (Cluster cluster = Cluster.builder()
-        .addContactPointsWithPorts(Collections.singleton(cassandra.contactPoint()))
-        .build(); Session session = TracingSession.create(withPropagation, cluster.connect())) {
-      session.execute(statement.apply(session));
+  static ImageFromDockerfile imageWithBraveTracer() {
```

Comment:
do you **really** need to build an image here? Why not just `withCopyFileToContainer`?

Reply:
didn't know about that :)